.buildkite/scripts/integration-tests.ps1

```diff
@@ -26,6 +26,10 @@ $env:SNAPSHOT = $true
 
 echo "~~~ Building test binaries"
 mage build:testBinaries
```

Comment:
```suggestion
& mage build:testBinaries
```
Helps be consistent and makes sure it doesn't happen to call some cmdlet named the same thing.